ui-tests/tst-resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for?

Reply:
Framework uses log4j2 as its logging system, this is configuring it to output to stdout in a nicer format and with more logging levels. Default settings for log4j is ERROR/WARN only if I remember correctly.